variables.tf

```
@@ -12,18 +12,22 @@ variable "prefix" {
   default     = "lacework"
   description = "The prefix to use at the beginning of every generated resource"
 }
+variable "application_id" {
+  type        = string
+  default     = ""
+  description = "The Active Directory Application id to use"
```

lets put the old description back please:
```suggestion
  description = "The Active Directory Application id to use. Required when `use_existing_ad_application` is set to `true`"```